src/server/models/users.model.js

```diff
@@ -8,6 +8,7 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
+  userId: String,
```

Comment:
Oh, I didn't consider mongodb crash or persistent...
This time I'll use _id instead of userId and I'll find out other systems about user_id!